### ISSUE TITLE
Deepseek-v3 SFT exhibits abnormal loss when dealing with long sequences.

### ISSUE THREAD

**Issue by cizhenshi** [alibaba/Pai-Megatron-Patch#560]
When fine-tuning Deepseek-v3 with longer sequences (16k), the initial loss tends to exceed 7. In the initial configuration file of Deepseek-v3, MAX_POSITION_EMBEDDINGS should be set to 4096 instead of 16k, because the position embedding scaling in Yarn is calculated based on MAX_POSITION_EMBEDDINGS. This value needs to match the initial setting during pre-training(4k). Additionally, in arguments.py, the restriction that MAX_POSITION_EMBEDDINGS must be greater than seq_length needs to be skipped .
```
#if args.seq_length is not None:
        #assert args.max_position_embeddings >= args.seq_length, \
        #    f"max_position_embeddings ({args.max_position_embeddings}) must be greater than " \
        #    f"or equal to seq_length ({args.seq_length})."
```
